dotcom-rendering/src/components/ElementContainer.tsx

```diff
@@ -116,14 +133,17 @@ export const ElementContainer = ({
 				 */
 				id={sectionId}
 				css={[
-					shouldCenter && center,
-					showSideBorders && sideBorderStyles(borderColour),
+					shouldCenter && !isTimeline && center,
```

Comment:
I'm not psyched about this. Open to suggestions!

Reply:
I think that overloading this logic inside the component slightly breaks the contract of the props interface, and I would consider whether we could ensure that setting `shouldCenter` and `showSideBorders` should be done where this component is called?

An alternative would be to [create a enumeration of all possible states](https://egghead.io/lessons/javascript-eliminate-boolean-explosion-by-enumerating-states), rather that having a combination of over half a dozen booleans.

Reply:
Thanks Max, much better. I've gone with the first option